### PR TITLE
Feature/onload

### PR DESCRIPTION
**Pull request by chilijung**
see https://github.com/FormidableLabs/victory-chart/pull/339

### CODE REVIEW

File: src/victory-util/transitions.js [hunk 1]
```diff
@@ -140,14 +181,15 @@ function getChildClipPathToExit(animate, child, data, exitingNodes, cb) { // esl
 
     if (beforeClipPathWidth) {
       clipWidth = beforeClipPathWidth(data, child, exitingNodes);
+      console.log(clipWidth)
```
Comment (boygirl) [inline src/victory-util/transitions.js]: Please lint

**Comment by boygirl** [comment]
@chilijung can you look into the failing tests on this branch. Also, I think the onExit clipPath bug is still a problem even after merging your other PR

**Comment by chilijung** [comment]
@boygirl Yeah, I've found out yesterday.  I'll get it fixed today.

**Comment by boygirl** [comment]
🎉 merging this now!

**Pull request merged**